python/src/aac/context/language_context.py

```diff
@@ -0,0 +1,896 @@
+        return self.parse_and_load(AAC_LANG_FILE_PATH)
+
+    def _get_aac_generated_class(self, name: str) -> Type:
+        definitions = self.get_definitions_by_name(name)
+        if len(definitions) != 1:
+            raise LanguageError(
+                f"_get_aac_generated_class unable to identify unique definition for name '{name}'.  Found: {[definition.name for definition in definitions]}"
+            )
+
+        aac_class = self.context_instance.fully_qualified_name_to_class[
+            definitions[0].get_fully_qualified_name()
+        ]
+        if not aac_class:
+            raise LanguageError(
+                f"_get_aac_generated_class unable to identify generated class for name '{name}' with fully_qualified_name '{definitions[0].get_fully_qualified_name()}'"
+            )
+
+        return aac_class
+
+    def is_aac_instance(self, obj: Any, name: str):
+        """Function to determine if an object is an instance of an AaC class."""
+        return isinstance(obj, self._get_aac_generated_class(name))
+
+    def create_aac_object(self, aac_type_name: str, attributes: dict) -> Any:
+        """Function to create a python instance of an AaC class and attributes."""
+        definitions = self.get_definitions_by_name(aac_type_name)
+        if len(definitions) != 1:
+            raise LanguageError(
+                f"Unable to identify unique definition for '{aac_type_name}'.  Found {[definition.name for definition in definitions]}"
+            )
+        aac_class = self._get_aac_generated_class(aac_type_name)
+        result = aac_class()
+        field_names = [field.name for field in definitions[0].instance.fields]
+        for attribute_name, attribute_value in attributes.items():
+            if attribute_name not in field_names:
+                raise LanguageError(f"Found undefined field name '{attribute_name}'")
+            setattr(result, attribute_name, attribute_value)
+        return result
+
+    def create_aac_enum(self, aac_enum_name: str, value: str) -> Any:
+        """Function to create a python instance of an AaC enum class and value."""
+        definitions = self.get_definitions_by_name(aac_enum_name)
+        if len(definitions) != 1:
+            raise LanguageError(
+                f"Unable to identify unique definition for '{aac_enum_name}'.  Found {[definition.name for definition in definitions]}"
+            )
+        aac_class = self._get_aac_generated_class(aac_enum_name)
+        try:
+            return getattr(aac_class, value)
+        except ValueError:
+            raise LanguageError(
+                f"{value} is not a valid value for enum {aac_enum_name}"
+            )
+
+    def parse_and_load(self, arg: str) -> list[Definition]:
+        """Convenience function that parses a file or string and loads the definitions into the context."""
+        parsed_definitions = parse(arg)
+
+        return self.load_definitions(parsed_definitions)
+
+    def load_definitions(  # noqa: C901
```

Comment:
I believe this method allows task #726 to be closed out.

The inner workings of this method is where the creation of the new POPO data object takes place. The documentation has been updated referencing the new data object and functionality of this method. And unit testing of the language context exists, including evaluating the functionality of this method and its results. Thus, meeting the outlined AC. 

Thoughts @bunchw , @crazynewidea , @cvchaparro ?

Reply:
I agree.

Reply:
Agree to close #726